readyplayerme/rpm-unity-sdk-core
# refactor and fix for loading user avatars

## [TICKETID](https://ready-player-me.atlassian.net/browse/TICKETID)

## Description

This PR includes:
- refactor of fetching user avatars
- reworked loading animation to use mecanim for better performance
- added support for hero (costume) outfits in avatar Creator examples
- updated template avatars (XR avatar now has all ARKit blendshapes






## How to Test

Test on new projects adding from this git url `https://github.com/readyplayerme/rpm-unity-sdk-core.git#hotfix/v6.3.2`

To test hero avatars you need:
1. to use a subdomain with hero outfits enabled (eg dev-sdk) 
2. run the CAC wizard scene
3. get to asset selection stage and then click on the new Star category (similar to web editor)
4. Select a hero asset

To test the updated user avatar fetching you should:
1. run the CAC wizard sample scene
2. Log in with an account that has some premade user avatars
3. after login you should see your user avatars loading
4. check for errors





## Checklist
